### PR TITLE
Make TVM build parallel in conda environment on Windows

### PR DESCRIPTION
**Pull request by kparzysz-quic**

### CODE REVIEW

File: conda/recipe/install_libtvm.bat [hunk 1]
```diff
@@ -15,7 +15,7 @@
 :: specific language governing permissions and limitations
 :: under the License.
 
-cmake --build build --config Release --target install
+cmake --build build --config Release --target install -- /p:CL_MPcount=8
```
Comment (driazati) [inline conda/recipe/install_libtvm.bat:18]: is it possible to use the windows equivalent of `nproc` here instead of always assuming it's `8`?

Comment (kparzysz-quic) [reply conda/recipe/install_libtvm.bat:18]: I think =0 will use all available processors.  I didn't see a lot of improvement from this (plus googling suggests limiting this number), so I'll try 2 and 4 next.

**Comment by kparzysz-quic** [comment]
There is really no difference in the Windows build time regardless of the value of this option.  :(

Closing this PR.

**Comment by driazati** [comment]
Just wondering is this for CI Windows builds? If so we could probably chase down #11550 to get some speedups

**Comment by kparzysz-quic** [comment]
Yes.  I thought it was an easy way to get some speedup, but it didn't work.  Builds with 2, 4, and 8 took about the same amount of time (roughly 100 minutes).  Also, many sources on the Internet suggested using a lower number, since multiple compilations could consume enough memory to cause extensive swapping.

The Windows build is not the longest in the CI, so I'm not sure if it's worth to spend a lot of effort trying to accelerate it.  It attracted my attention because I had to fix build failures on Windows (related to the switch to C++17), and each round took quite a while to get to the point of failure.  On the other hand, if you have time, don't let anything dissuade you from working on this.  🙂

**Pull request closed without merging**